conbench/templates/benchmark-entity.html

```diff
@@ -65,7 +65,8 @@
             {% for k,v in benchmark.error.items() %}
               <li class="list-group-item" style="overflow-y: auto;">
                 <b>{{ k }}</b>
-                <div align="right" style="display:inline-block; float: right;">{{ v }}</div>
+                <br></br>
```

Comment:
```suggestion
                 <br />
```

One last, minor suggestion